optimize/backend/src/main/java/io/camunda/optimize/tomcat/ResponseHeadersFilter.java

```diff
@@ -0,0 +1,44 @@
+package io.camunda.optimize.tomcat;
```

Comment:
I am always chaining this filter.
I don't have a lot of context about these headers. If you think this only applies to a subset of the cases, please feel free to let me know.